src/pages/instances/actions/CreateImageFromInstanceBtn.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { ActionButton } from "@canonical/react-components";
+import usePortal from "react-useportal";
+import CreateImageFromInstanceForm from "../forms/CreateImageFromInstanceForm";
+import { useInstanceLoading } from "context/instanceLoading";
+
+interface Props {
+  instance: LxdInstance;
+}
+
+const CreateImageFromInstanceBtn: FC<Props> = ({ instance }) => {
+  const { openPortal, closePortal, isOpen, Portal } = usePortal();
+  const instanceLoading = useInstanceLoading();
+  const prohibitedStatuses = ["Error", "Frozen", "Running"];
+
+  const isDisabled = (): boolean => {
+    if (
+      prohibitedStatuses.includes(instance?.status) ||
+      Boolean(instanceLoading.getType(instance))
+    ) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
```suggestion
  const isDisabled = prohibitedStatuses.includes(instance?.status) ||
      Boolean(instanceLoading.getType(instance))
```
No need for a function here I think.